<launch>
  <group ns="warehouse">
    <group ns="raws">
      <node pkg="robonomics_game_warehouse" name="yellow" type="warehouse_node.py">
        <param name="size" value="[3, 4]" />
        <param name="offset" value="[0, 0]" />
        <param name="warehouse" value="raws" />
        <param name="content" value="yellow" />
      </node>
      <node pkg="robonomics_game_warehouse" name="green" type="warehouse_node.py">
        <param name="size" value="[3, 4]" />
        <param name="offset" value="[3, 0]" />
        <param name="warehouse" value="raws" />
        <param name="content" value="green" />
      </node>
      <node pkg="robonomics_game_warehouse" name="blue" type="warehouse_node.py">
        <param name="size" value="[3, 4]" />
        <param name="offset" value="[6, 0]" />
        <param name="warehouse" value="raws" />
        <param name="content" value="blue" />
      </node>
      <node pkg="robonomics_game_warehouse" name="purple" type="warehouse_node.py">
        <param name="size" value="[3, 4]" />
        <param name="offset" value="[9, 0]" />
        <param name="warehouse" value="raws" />
        <param name="content" value="purple" />
      </node>
    </group>
  </group>
  <param name="opcua_client_node" value="/opcua/opcua_client" />
  <group ns="transport_raws">
    <node pkg="robonomics_game_transport" name="stacker_raws" type="stacker_node.py">
      <param name="~opcua_server_namespace" value="ns=3;s=/Airalab/StackerRaws" />
      <param name="direction" value="forward" />
      <param name="timeout" value="5000" />
    </node>
    <node pkg="robonomics_game_transport" name="supplier" type="supplier_node.py">
      <param name="stacker_node" value="/transport_raws/stacker_raws" />
    </node>
  </group>
</launch>
